public/app/plugins/datasource/cloudwatch/components/__snapshots__/QueryEditor.test.tsx.snap

```diff
@@ -1,369 +1,3 @@
-  </div>,
-  <div
-    className="gf-form-inline"
-  >
-    <div
-      className="gf-form"
-    >
-      <label
-        className="gf-form-label width-8 query-keyword"
-      >
-        Period
-        <div
-          className="gf-form-help-icon gf-form-help-icon--right-normal"
-          onMouseEnter={[Function]}
-          onMouseLeave={[Function]}
-        >
-          <i
-            className="fa fa-info-circle"
-          />
-        </div>
-      </label>
-      <div
-        style={
-          Object {
-            "flexGrow": 1,
-          }
-        }
-      >
-        <input
-          className="gf-form-input gf-form-input width-8"
-          onBlur={[MockFunction]}
-          onChange={[Function]}
-          placeholder="auto"
-          value=""
-        />
-      </div>
-    </div>
-    <div
-      className="gf-form"
-    >
-      <label
-        className="gf-form-label width-8 query-keyword"
-      >
-        Alias
-        <div
-          className="gf-form-help-icon gf-form-help-icon--right-normal"
-          onMouseEnter={[Function]}
-          onMouseLeave={[Function]}
-        >
-          <i
-            className="fa fa-info-circle"
-          />
-        </div>
-      </label>
-      <div
-        style={
-          Object {
-            "flexGrow": 1,
-          }
-        }
-      >
-        <input
-          className="gf-form-input gf-form-input width-16"
-          onChange={[Function]}
-          type="text"
-          value=""
-        />
-      </div>
-      <div
-        className="gf-form-switch-container-react"
-      >
-        <label
-          className="gf-form gf-form-switch-container "
-          htmlFor="1"
-        >
-          <div
-            className="gf-form-label query-keyword  pointer"
-          >
-            Match Exact
-            <div
-              className="gf-form-help-icon gf-form-help-icon--right-normal"
-              onMouseEnter={[Function]}
-              onMouseLeave={[Function]}
-            >
-              <i
-                className="fa fa-info-circle"
-              />
-            </div>
-          </div>
-          <div
-            className="gf-form-switch  "
-          >
-            <input
-              checked={true}
-              id="1"
-              onChange={[Function]}
-              type="checkbox"
-            />
-            <span
-              className="gf-form-switch__slider"
-            />
-          </div>
-        </label>
-      </div>
-      <label
-        className="gf-form-label"
-      >
-        <a
-          onClick={[Function]}
-        >
-          <i
-            className="fa fa-caret-right"
-          />
-           
-          Show
-           Query Preview
-        </a>
-      </label>
-    </div>
-    <div
-      className="gf-form gf-form--grow"
-    >
-      <div
-        className="gf-form-label gf-form-label--grow"
-      />
-    </div>
-  </div>,
-]
-`;
+exports[`QueryEditor should render component 1`] = `null`;
```

Comment:
Just a quick question, why instead of rendering component it renders 'null'?

Reply:
@ivanahuckova Looks like it's because of wrapping it with `act()`, that's there due to async state updates in components. Without that we get a component rendered, but we also get an error about state updates.